ui/src/app/modules/status/widgets/update-info-widget/update-info-widget.component.ts

```diff
@@ -178,4 +197,39 @@ export class UpdateInfoWidgetComponent implements OnInit {
       this.$toastr.error(error.message, this.$translate.instant('toast.title_error'))
     }
   }
+
+  private async getDockerInfo() {
+    if (!this.serverInfo?.homebridgeRunningInDocker) {
+      this.dockerStatusDone = true
+    } else {
+      try {
+        this.dockerInfo = await firstValueFrom(this.io.request('docker-version-check'))
+        this.dockerStatusDone = true
+      } catch (error) {
+        console.error(error)
+        this.$toastr.error(error.message, this.$translate.instant('toast.title_error'))
+      }
+    }
```

Comment:
i personally prefer not having the negation in the if, so that the else isn't like a double negative.

so like

```
if (this.serverInfo?.homebridgeRunningInDocker) {
  // ...
} else {
  this.dockerStatusDone = true
}
```

but this is just my preference, and if it is not changed here then i will probably end up doing it at a later date anyway 😄